testing/getmail/APKBUILD

```diff
@@ -1,31 +1,28 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=getmail
-pkgver=5.8
+pkgver=5.13
 pkgrel=0
 pkgdesc="POP3, IMAP4, and SDPS mail retriever with Maildir delivery"
 url="http://pyropus.ca/software/getmail/"
 arch="noarch"
-license="GPL-2.0"
+license="GPL-2.0-only"
 depends="python2"
-depends_dev=""
 makedepends="python2-dev"
-install=""
 subpackages="$pkgname-doc"
 options="!check" # contains no tests
 source="http://pyropus.ca/software/getmail/old-versions/$pkgname-$pkgver.tar.gz"
 builddir="$srcdir"/$pkgname-$pkgver
 
 build() {
-	cd "$builddir"
```

Comment:
No reason to remove

Reply:
@andypost I've kinda been telling that `cd "$builddir"` is redundant nowadays..

Reply:
ncopa asked to keep them at least in main (https://github.com/alpinelinux/aports/pull/7244)

If you want i can expand the keeping 'cd "$builddir"' to all repos.